src/gfx/reverse.cpp

```diff
@@ -366,9 +370,8 @@ void reverse() {
 			    0x00,
 			    0x00,
 			};
-			uint8_t const *tileData = tileID > nbTileInstances - options.trim
-			                              ? trimmedTile.data()
-			                              : &tiles[tileID * tileSize];
+			uint8_t const *tileData =
+			    tileID > nbTiles ? trimmedTile.data() : &tiles[tileID * tileSize];
```

Comment:
Umm, shouldn't this be

```suggestion
			    tileID >= nbTiles ? trimmedTile.data() : &tiles[tileID * tileSize];
```

actually? If yes, then looks like we've never tested this, lol.